tests/unit/form-fillna-step.spec.ts

```diff
@@ -0,0 +1,162 @@
+import { mount, shallowMount, createLocalVue } from '@vue/test-utils';
+import FormFillnaStep from '@/components/FormFillnaStep.vue';
+import WidgetAutocomplete from '@/components/WidgetAutocomplete.vue';
+import Vuex, { Store } from 'vuex';
+import { setupStore } from '@/store';
+import { Pipeline } from '@/lib/steps';
+import { VQBState } from '@/store/state';
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+interface ValidationError {
```

Comment:
I think that you can use the type ajv's definition `ErrorObject` instead of creating a new one